sections/sprint-4/client-side-handling.tex

```diff
@@ -5,7 +5,18 @@ \section{Receiving the information}
 
 \subsection*{UDP client}
 Handling the datagrams on the UDP client is simple as it can currently only receive a message with type 0, which is to a message containing a position for a Pozyx tag.
-When this datagram is received, we convert the data to a byte and switch on it as seen on \autoref{lst:readingudpdatagram}.
+The datagram is recieved in the format \textit{0xYYYYXXXXIISSTT} where:
+\begin{itemize}
+    \item Y - Y position of tag
+    \item X - X position of tag
+    \item I - Id of player
+    \item S - Timestamp
+    \item T - Package type
+\end{itemize}
+
+When this datagram is received, we start by parsing the hexadecimal to a long.
+Because one hexadecimal corresponds to four bits we can get type of the datagram which is the last two hexadecimals of the datagram by typecasting it to a byte.
```

Comment:
```suggestion
Since one hexadecimal corresponds to four bits we can get the type of the message, which is the last two hex values of the datagram by typecasting it to a byte.
```